### PR TITLE
feat(preset-mini): support using kebab-case for fontSize

### PR DESCRIPTION
**Pull request by Skiyee**
Using kebab-case just like a color theme.

```
theme: {
  fontSize: {
    bodySmall: '1rem',
    titleMedium: ['2rem', '1.5em'],
  }
}
```

before: `text-bodySmall`
now: `text-body-small` | `text-body-small:15em`

### CODE REVIEW

File: test/preset-mini.test.ts [hunk 1]
```diff
@@ -443,4 +443,33 @@ describe('preset-mini', () => {
     .border-opacity-50{--un-border-opacity:0.5;}"
   `)
   })
+
+  it('use fontSize by camel-case', async () => {
```
[the remark this answers is not in the document]
Comment (onmax) [reply test/preset-mini.test.ts:447]: "Support for font size handling with camel case notation" 🤔

**Comment by antfu** [comment]
Emm, no, I don't think we should support uppercase or this convention. It shouldn't be hard to use quoted keys like:

```ts
  fontSize: {
    'body-small': '1rem',
  }
```

**Comment by Skiyee** [comment]
All right. ❤
I just want it to be like a color theme, with two ways to use the naming.

**Comment by Skiyee** [comment]
@antfu 
请求Open，我现在需要做转换才能统一风格

若坚持Close，希望你们可以让 fontSize 跟 colors 一样在主题内配置单驼峰时，也接受 kebab-case

**Pull request closed without merging**